### PR TITLE
Trend Micro Cloud One - Conformity: chore

### PR DESCRIPTION
**Pull request by rohitkaul83**
## Summary

add description for documentation, add keywords, shorten title

### CODE REVIEW

File: packs/trendmicro-cloudone-conformity/config.yml [hunk 1]
```diff
@@ -28,8 +28,18 @@ authors:
...
     url: https://newrelic.com/blog/how-to-relic/trend-micro-aws
   - name: Trend Micro blog post
+    description: description: Learn more about the integration on Trend Micro's blog post
     url: https://www.trendmicro.com/en_us/devops/21/f/how-to-achieve-more-security-observability.html
+
+keywords:
+  - cloud
+  - devops
+  - aws
+  - trend micro
```
Comment (jpvajda) [inline packs/trendmicro-cloudone-conformity/config.yml:44]:
```suggestion

```

File: packs/trendmicro-cloudone-conformity/config.yml [hunk 2]
```diff
@@ -28,8 +28,18 @@ authors:
...
     url: https://github.com/newrelic-experimental/newrelic-connector-trendmicro-cloud-one-conformity#readme
   - name: New Relic blog post
+    description: Learn more about the integration on New Relic's blog post
     url: https://newrelic.com/blog/how-to-relic/trend-micro-aws
   - name: Trend Micro blog post
+    description: description: Learn more about the integration on Trend Micro's blog post
     url: https://www.trendmicro.com/en_us/devops/21/f/how-to-achieve-more-security-observability.html
+
+keywords:
+  - cloud
```
Comment (jpvajda) [inline packs/trendmicro-cloudone-conformity/config.yml:41]:
```suggestion

```

**Review by jpvajda — CHANGES_REQUESTED** [review]
I removed some keywords that aren't approved.

**Comment by jpvajda** [comment]
most of the checks are failing on this PR so we'll have to see what is occurring here. @aswanson-nr

File: packs/trendmicro-cloudone-conformity/config.yml [hunk 3]
```diff
@@ -28,8 +28,18 @@ authors:
 
 documentation:
   - name: GitHub repository
+    description: The GitHub repository for the integration
     url: https://github.com/newrelic-experimental/newrelic-connector-trendmicro-cloud-one-conformity#readme
   - name: New Relic blog post
+    description: Learn more about the integration on New Relic's blog post
     url: https://newrelic.com/blog/how-to-relic/trend-micro-aws
   - name: Trend Micro blog post
+    description: description: Learn more about the integration on Trend Micro's blog post
```
Comment (aswanson-nr) [inline packs/trendmicro-cloudone-conformity/config.yml:37]: The extra `description:` here is invalidating the yaml when the CI checks attempt to read it. 
```suggestion
    description: Learn more about the integration on Trend Micro's blog post
```

File: packs/trendmicro-cloudone-conformity/config.yml (hunk 1 above)
Comment (jpvajda) [reply packs/trendmicro-cloudone-conformity/config.yml:43]: I'm going to let this one be used, as we are establishing a pattern where a company name could make a good keyword in other packs.

**Review by jpvajda — APPROVED** [review]

**Pull request merged**